src/channel/set.ts

```diff
@@ -115,7 +115,7 @@ export async function setChannel(channel: string, appId: string, options: Option
     }
     if (downgrade != null) {
       p.log.info(`Set ${appId} channel: ${channel} to ${downgrade ? 'allow' : 'disallow'} downgrade`)
-      channelPayload.disableAutoUpdateUnderNative = !downgrade
```

Comment:
Can we find a way to make CLI compatible with old and new?
The goal will be to merge this first and next week the backend to minimize breaking users